scylla/README.md

```diff
@@ -0,0 +1,51 @@
+# Agent Check: Scylla
+
+## Overview
+
+This check monitors [Scylla][1] through the Datadog Agent.
+
+## Setup
+
+Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][2] for guidance on applying these instructions.
```

Comment:
Can we add an `auto_conf.yaml` if scylla integration supports autodiscovery?

Reply:
Hmm, this was a leftover from the previous README, I don't think Autodiscovery really applies to this integration.

Reply:
Then we can just include host setup instructions here. https://github.com/DataDog/integrations-core/tree/master/mysql#host